README.md

```diff
@@ -52,23 +65,25 @@ Exit the server:
 exit
 ```
 
-## Usage
+# Usage-of-myProxy-Website
 
-Go to server URL:
-```
-http://your-server-ip-address:3000
-```
+Sign in using admin password and use provider's API key and secret to setup domains:
 
-You will be prompted to enter your admin password and your domain provider's API Key and Secret, [find out how here](https://github.com/Neilpang/acme.sh/wiki/dnsapi)
+1. Go to server URL: ```http://my\_server\_ip:3000```
+2. Sign in with my\_admin\_password to gain access to admin page
```

Comment:
this should not be hardcoded. `Sign in with the password you set when you ran the app to gain access to admin page`